src/main/java/org/folio/circulation/resources/ItemsInTransitResource.java

```diff
@@ -72,12 +75,13 @@ private void getMany(RoutingContext routingContext) {
     final ReportRepository reportRepository = new ReportRepository(clients);
     final UserRepository userRepository = new UserRepository(clients);
     final PatronGroupRepository patronGroupRepository = new PatronGroupRepository(clients);
+    final Comparator<InTransitReportEntry> sortByCheckinServicePointComparator = sortByCheckinServicePointComparator();
 
     reportRepository.getAllItemsByField("status.name", IN_TRANSIT.getValue())
       .thenComposeAsync(r -> r.after(itemsReportFetcher ->
         fetchItemsRelatedRecords(itemsReportFetcher, itemRepository, servicePointRepository)))
       .thenComposeAsync(r -> r.after(inTransitReportEntries ->
-        fetchLoans(loansStorageClient, servicePointRepository, inTransitReportEntries)))
+        fetchLoans(loansStorageClient, servicePointRepository, inTransitReportEntries, sortByCheckinServicePointComparator)))
```

Comment:
These lines that use the comparator are pretty long, please split onto a second line if well over 80 characters

Reply:
Done